salt/elasticsearch/files/config/readonlyrest-docs.yml

```diff
@@ -2,9 +2,8 @@ readonlyrest:
   # https://github.com/beshu-tech/readonlyrest-docs/blob/master/kibana.md#loading-settings-order-of-precedence
   force_load_from_file: True
   ssl:
-    keystore_file: keystore.jks
-    keystore_pass: {{ pillar.elasticsearch.plugins.readonlyrest.key_pass }}
-    key_pass: {{ pillar.elasticsearch.plugins.readonlyrest.key_pass }}
+    server_certificate_key_file: /etc/apache2/md/domains/standard.open-contracting.org/privkey.pem
```

Comment:
I don't think this should be hard-coded, perhaps a variable like: ``elasticsearch:plugins:readonlyrest:privkey``

Reply:
Sounds good to me.

Reply:
Done ✔